src/components/LikeButton/Likes.jsx

```diff
@@ -0,0 +1,29 @@
+import { useState } from "react";
+import { updateVotes } from "../../utils";
+import './Likes.css'
+
+
+export default function LikeCount({article}) {
+  const [votes, setVotes] = useState(article.votes) 
+
+
+  function handleClick(event) {
+    event.preventDefault();
+    setVotes((currentVotes) => currentVotes + 1)
+    updateVotes(article.article_id)
```

Comment:
You've done half of optimistic rendering which is good 👍 
However when we render optimistically we need to make sure we correct the state if the API request happens to fail
I.e. if the internet fails when we're pressing the button, can we revert the votes so the user knows it's hasn't gone through
Notes for more info: https://notes.northcoders.com/courses/js-front-end/ui-behaviours-and-optimistic-rendering